src/components/Table/index.js

```diff
@@ -7,6 +7,9 @@ import {
   Badge,
   NavLink,
   Button,
+  Modal,
+  ModalHeader,
+  ModalFooter,
```

Comment:
Love how you investigated what library we've used for components and introduced the `Modal` component here - thank you!